msal/src/main/java/com/microsoft/identity/client/internal/AsyncResult.java

```diff
@@ -29,26 +29,21 @@
 public class AsyncResult<T> {
     private T mResult;
     private MsalException mMsalException;
-    private boolean mSuccess = false;
 
-    public AsyncResult(@Nullable T result, @Nullable MsalException exception){
+    public AsyncResult(@Nullable T result, @Nullable MsalException exception) {
         mResult = result;
         mMsalException = exception;
```

Comment:
Optionally, we could consider throwing a RuntimeException if both params are `null` (as this would be a bug and should never happen)

Reply:
Isn't this what we discussed yesterday and we concluded that we would allow both to be `null` as is the case for the `getAccount(identifier)` API? (it returns `null` without exception if an account for the identifier doesn't exist).

Reply:
🤦‍♂️ 

Duh!

Reply:
Long day 😆